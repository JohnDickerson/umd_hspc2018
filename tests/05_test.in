6
10
0 3
1 6
2 2
3 5
5 6
6 1
7 4
8 2
10 1
11 2
12 5
16
0 0
1 6
2 2
4 7
5 5
6 6
7 0
8 1
9 2
10 -2
11 3
12 4
13 7
14 1
15 5
16 -1
17 3
28
0 0
1 6
2 2
4 7
5 5
6 6
7 0
8 1
9 2
10 -2
11 3
12 4
13 7
14 1
15 5
16 -1
17 3
18 1
19 7
20 5
22 7
23 2
24 6
25 7
26 5
27 6
28 0
29 1
30 4
56
0 0
1 9
2 12
3 13
4 4
5 11
6 10
7 13
8 14
9 11
10 10
11 12
12 12
14 0
15 4
16 2
17 2
18 4
19 3
20 -4
21 -1
22 6
23 8
24 8
25 6
26 14
27 12
28 2
29 12
30 10
32 -2
33 5
34 6
35 3
36 2
37 -1
38 14
39 11
40 10
41 9
43 8
44 14
45 12
46 4
47 11
48 12
49 11
50 14
51 13
52 10
53 9
54 12
56 0
57 2
58 2
59 6
60 8
1
0 0
1 0
200
0 89
1 46
2 45
3 54
4 84
5 69
6 65
7 82
8 17
9 90
10 64
11 29
12 63
13 53
14 63
15 12
16 41
17 26
18 25
19 4
20 79
21 50
22 96
23 17
24 90
25 32
26 94
27 37
28 41
29 24
30 65
31 37
32 99
33 67
34 53
35 60
36 51
37 28
38 26
39 54
40 79
41 71
42 17
43 73
44 38
45 93
46 26
47 56
48 77
49 0
50 52
51 81
52 47
53 13
54 98
55 45
56 14
57 0
58 47
59 25
60 10
61 71
62 15
63 72
64 86
65 40
66 27
67 20
68 98
69 45
70 23
71 27
72 10
73 1
74 6
75 23
76 33
77 5
78 31
79 10
80 3
81 52
82 40
83 1
84 66
85 57
86 71
87 7
88 21
89 80
90 66
91 30
92 98
93 28
94 83
95 2
96 56
97 52
98 89
99 49
100 41
101 88
102 71
103 12
104 38
105 11
106 83
107 1
108 71
109 47
110 36
111 37
112 83
113 57
114 9
115 41
116 3
117 80
118 37
119 88
120 35
121 49
122 47
123 72
124 65
125 85
126 86
127 79
128 47
129 23
130 7
131 85
132 97
133 28
134 4
135 13
136 88
137 77
138 97
139 80
140 90
141 20
142 42
143 81
144 33
145 12
146 83
147 24
148 55
149 55
150 93
151 14
152 44
153 33
154 78
155 95
156 32
157 3
158 28
159 51
160 13
161 14
162 79
163 17
164 2
165 27
166 20
167 30
168 70
169 24
170 19
171 35
172 46
173 73
174 21
175 71
176 9
177 96
178 63
179 63
180 70
181 58
182 43
183 3
184 65
185 32
186 16
187 14
188 15
189 99
190 33
191 19
192 44
193 88
194 67
195 76
196 27
197 3
198 36
199 55
200 77
