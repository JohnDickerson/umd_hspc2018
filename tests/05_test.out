Trial: 1
  Points:
    [0]: (0,3)
    [1]: (1,6)
    [2]: (2,2)
    [3]: (3,5)
    [4]: (5,6)
    [5]: (6,1)
    [6]: (7,4)
    [7]: (8,2)
    [8]: (10,1)
    [9]: (11,2)
    [10]: (12,5)
  Tour: 0 1 3 4 6 10 9 8 7 5 2 0
  Total distance = 40
Trial: 2
  Points:
    [0]: (0,0)
    [1]: (1,6)
    [2]: (2,2)
    [3]: (4,7)
    [4]: (5,5)
    [5]: (6,6)
    [6]: (7,0)
    [7]: (8,1)
    [8]: (9,2)
    [9]: (10,-2)
    [10]: (11,3)
    [11]: (12,4)
    [12]: (13,7)
    [13]: (14,1)
    [14]: (15,5)
    [15]: (16,-1)
    [16]: (17,3)
  Tour: 0 1 3 4 5 10 11 12 14 16 15 13 9 8 7 6 2 0
  Total distance = 74
Trial: 3
  Points:
    [0]: (0,0)
    [1]: (1,6)
    [2]: (2,2)
    [3]: (4,7)
    [4]: (5,5)
    [5]: (6,6)
    [6]: (7,0)
    [7]: (8,1)
    [8]: (9,2)
    [9]: (10,-2)
    [10]: (11,3)
    [11]: (12,4)
    [12]: (13,7)
    [13]: (14,1)
    [14]: (15,5)
    [15]: (16,-1)
    [16]: (17,3)
    [17]: (18,1)
    [18]: (19,7)
    [19]: (20,5)
    [20]: (22,7)
    [21]: (23,2)
    [22]: (24,6)
    [23]: (25,7)
    [24]: (26,5)
    [25]: (27,6)
    [26]: (28,0)
    [27]: (29,1)
    [28]: (30,4)
  Tour: 0 1 3 4 5 10 11 12 14 18 19 20 22 23 24 25 28 27 26 21 17 16 15 13 9 8 7 6 2 0
  Total distance = 120
Trial: 4
  Points:
    [0]: (0,0)
    [1]: (1,9)
    [2]: (2,12)
    [3]: (3,13)
    [4]: (4,4)
    [5]: (5,11)
    [6]: (6,10)
    [7]: (7,13)
    [8]: (8,14)
    [9]: (9,11)
    [10]: (10,10)
    [11]: (11,12)
    [12]: (12,12)
    [13]: (14,0)
    [14]: (15,4)
    [15]: (16,2)
    [16]: (17,2)
    [17]: (18,4)
    [18]: (19,3)
    [19]: (20,-4)
    [20]: (21,-1)
    [21]: (22,6)
    [22]: (23,8)
    [23]: (24,8)
    [24]: (25,6)
    [25]: (26,14)
    [26]: (27,12)
    [27]: (28,2)
    [28]: (29,12)
    [29]: (30,10)
    [30]: (32,-2)
    [31]: (33,5)
    [32]: (34,6)
    [33]: (35,3)
    [34]: (36,2)
    [35]: (37,-1)
    [36]: (38,14)
    [37]: (39,11)
    [38]: (40,10)
    [39]: (41,9)
    [40]: (43,8)
    [41]: (44,14)
    [42]: (45,12)
    [43]: (46,4)
    [44]: (47,11)
    [45]: (48,12)
    [46]: (49,11)
    [47]: (50,14)
    [48]: (51,13)
    [49]: (52,10)
    [50]: (53,9)
    [51]: (54,12)
    [52]: (56,0)
    [53]: (57,2)
    [54]: (58,2)
    [55]: (59,6)
    [56]: (60,8)
  Tour: 0 1 2 3 5 6 7 8 9 10 11 12 25 26 28 29 36 37 38 39 40 41 42 44 45 46 47 48 49 50 51 56 55 54 53 52 43 35 34 33 32 31 30 27 24 23 22 21 20 19 18 17 16 15 14 13 4 0
  Total distance = 266
Trial: 5
  Points:
    [0]: (0,0)
    [1]: (1,0)
  Tour: 0 1 0
  Total distance = 2
Trial: 6
  Points:
    [0]: (0,89)
    [1]: (1,46)
    [2]: (2,45)
    [3]: (3,54)
    [4]: (4,84)
    [5]: (5,69)
    [6]: (6,65)
    [7]: (7,82)
    [8]: (8,17)
    [9]: (9,90)
    [10]: (10,64)
    [11]: (11,29)
    [12]: (12,63)
    [13]: (13,53)
    [14]: (14,63)
    [15]: (15,12)
    [16]: (16,41)
    [17]: (17,26)
    [18]: (18,25)
    [19]: (19,4)
    [20]: (20,79)
    [21]: (21,50)
    [22]: (22,96)
    [23]: (23,17)
    [24]: (24,90)
    [25]: (25,32)
    [26]: (26,94)
    [27]: (27,37)
    [28]: (28,41)
    [29]: (29,24)
    [30]: (30,65)
    [31]: (31,37)
    [32]: (32,99)
    [33]: (33,67)
    [34]: (34,53)
    [35]: (35,60)
    [36]: (36,51)
    [37]: (37,28)
    [38]: (38,26)
    [39]: (39,54)
    [40]: (40,79)
    [41]: (41,71)
    [42]: (42,17)
    [43]: (43,73)
    [44]: (44,38)
    [45]: (45,93)
    [46]: (46,26)
    [47]: (47,56)
    [48]: (48,77)
    [49]: (49,0)
    [50]: (50,52)
    [51]: (51,81)
    [52]: (52,47)
    [53]: (53,13)
    [54]: (54,98)
    [55]: (55,45)
    [56]: (56,14)
    [57]: (57,0)
    [58]: (58,47)
    [59]: (59,25)
    [60]: (60,10)
    [61]: (61,71)
    [62]: (62,15)
    [63]: (63,72)
    [64]: (64,86)
    [65]: (65,40)
    [66]: (66,27)
    [67]: (67,20)
    [68]: (68,98)
    [69]: (69,45)
    [70]: (70,23)
    [71]: (71,27)
    [72]: (72,10)
    [73]: (73,1)
    [74]: (74,6)
    [75]: (75,23)
    [76]: (76,33)
    [77]: (77,5)
    [78]: (78,31)
    [79]: (79,10)
    [80]: (80,3)
    [81]: (81,52)
    [82]: (82,40)
    [83]: (83,1)
    [84]: (84,66)
    [85]: (85,57)
    [86]: (86,71)
    [87]: (87,7)
    [88]: (88,21)
    [89]: (89,80)
    [90]: (90,66)
    [91]: (91,30)
    [92]: (92,98)
    [93]: (93,28)
    [94]: (94,83)
    [95]: (95,2)
    [96]: (96,56)
    [97]: (97,52)
    [98]: (98,89)
    [99]: (99,49)
    [100]: (100,41)
    [101]: (101,88)
    [102]: (102,71)
    [103]: (103,12)
    [104]: (104,38)
    [105]: (105,11)
    [106]: (106,83)
    [107]: (107,1)
    [108]: (108,71)
    [109]: (109,47)
    [110]: (110,36)
    [111]: (111,37)
    [112]: (112,83)
    [113]: (113,57)
    [114]: (114,9)
    [115]: (115,41)
    [116]: (116,3)
    [117]: (117,80)
    [118]: (118,37)
    [119]: (119,88)
    [120]: (120,35)
    [121]: (121,49)
    [122]: (122,47)
    [123]: (123,72)
    [124]: (124,65)
    [125]: (125,85)
    [126]: (126,86)
    [127]: (127,79)
    [128]: (128,47)
    [129]: (129,23)
    [130]: (130,7)
    [131]: (131,85)
    [132]: (132,97)
    [133]: (133,28)
    [134]: (134,4)
    [135]: (135,13)
    [136]: (136,88)
    [137]: (137,77)
    [138]: (138,97)
    [139]: (139,80)
    [140]: (140,90)
    [141]: (141,20)
    [142]: (142,42)
    [143]: (143,81)
    [144]: (144,33)
    [145]: (145,12)
    [146]: (146,83)
    [147]: (147,24)
    [148]: (148,55)
    [149]: (149,55)
    [150]: (150,93)
    [151]: (151,14)
    [152]: (152,44)
    [153]: (153,33)
    [154]: (154,78)
    [155]: (155,95)
    [156]: (156,32)
    [157]: (157,3)
    [158]: (158,28)
    [159]: (159,51)
    [160]: (160,13)
    [161]: (161,14)
    [162]: (162,79)
    [163]: (163,17)
    [164]: (164,2)
    [165]: (165,27)
    [166]: (166,20)
    [167]: (167,30)
    [168]: (168,70)
    [169]: (169,24)
    [170]: (170,19)
    [171]: (171,35)
    [172]: (172,46)
    [173]: (173,73)
    [174]: (174,21)
    [175]: (175,71)
    [176]: (176,9)
    [177]: (177,96)
    [178]: (178,63)
    [179]: (179,63)
    [180]: (180,70)
    [181]: (181,58)
    [182]: (182,43)
    [183]: (183,3)
    [184]: (184,65)
    [185]: (185,32)
    [186]: (186,16)
    [187]: (187,14)
    [188]: (188,15)
    [189]: (189,99)
    [190]: (190,33)
    [191]: (191,19)
    [192]: (192,44)
    [193]: (193,88)
    [194]: (194,67)
    [195]: (195,76)
    [196]: (196,27)
    [197]: (197,3)
    [198]: (198,36)
    [199]: (199,55)
    [200]: (200,77)
  Tour: 0 1 2 3 5 6 8 11 15 16 17 18 19 23 25 27 28 29 30 31 37 38 42 44 46 49 50 52 53 56 57 59 60 62 65 66 67 70 75 76 78 81 82 84 85 86 89 90 92 94 98 101 102 106 108 109 110 111 112 113 115 117 119 123 124 125 126 127 131 132 136 137 138 139 140 143 146 148 149 150 154 155 162 168 173 175 177 178 179 180 181 182 184 189 193 194 195 200 199 198 197 196 192 191 190 188 187 186 185 183 176 174 172 171 170 169 167 166 165 164 163 161 160 159 158 157 156 153 152 151 147 145 144 142 141 135 134 133 130 129 128 122 121 120 118 116 114 107 105 104 103 100 99 97 96 95 93 91 88 87 83 80 79 77 74 73 72 71 69 68 64 63 61 58 55 54 51 48 47 45 43 41 40 39 36 35 34 33 32 26 24 22 21 20 14 13 12 10 9 7 4 0
  Total distance = 3432
